openedx/features/badging/tests/test_helpers.py

```diff
@@ -1,23 +1,19 @@
 import factory
 import mock
-
 from django.db.models import signals
 
 from lms.djangoapps.teams.tests.factories import CourseTeamFactory, CourseTeamMembershipFactory
-from nodebb.constants import (
-    TEAM_PLAYER_ENTRY_INDEX,
-    CONVERSATIONALIST_ENTRY_INDEX
-)
+from nodebb.constants import CONVERSATIONALIST_ENTRY_INDEX, TEAM_PLAYER_ENTRY_INDEX
 from openedx.features.badging.constants import CONVERSATIONALIST, TEAM_PLAYER
 from openedx.features.teams.tests.factories import TeamGroupChatFactory
-from student.tests.factories import CourseEnrollmentFactory
-from student.tests.factories import UserFactory
+from student.tests.factories import CourseEnrollmentFactory, UserFactory
 from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
 from xmodule.modulestore.tests.factories import CourseFactory
 
-from .factories import BadgeFactory, UserBadgeFactory
 from .. import helpers as badge_helpers
 from ..models import Badge
+from .factories import BadgeFactory, UserBadgeFactory
+from .. import handlers as badge_handlers
```

Comment:
Lets not import all handlers, I think we should import `register_notification_types` only

Reply:
done